### ISSUE TITLE
A11y_SQL database projects extension_Database Project_Database Project_Name Role Value: Incorrect role is defined for the 'read our docs' link as button.

### ISSUE THREAD

**Issue by Samhitha66** [microsoft/azuredatastudio#22096]
### GitHub Tags:

#A11yMAS; #A11yTCS; #A11ySev3; #DesktopApp; #Name Role Value; #WCAG4.1.2; #Win11; #SH-SQL Azure Data Studio-Win32-March23; #Win32; #SQL Azure Data Studio; #AILimited;

"[Check out Accessibility Insights!](https://nam06.safelinks.protection.outlook.com/?url=https://accessibilityinsights.io/&data=02%7c01%7cv-stfe%40microsoft.com%7cb67b2c4b646d4f9561a208d6f4b5c39b%7c72f988bf86f141af91ab2d7cd011db47%7c1%7c0%7c636965458850501301&sdata=mxhokIKNMb22llsjXHLgU3XZibj1Qfx37rpY4PU2sfE%3D&reserved=0) - Identify accessibility bugs before check-in and make bug fixing faster and easier."

### Environment Details:​
Application Name: SQL Azure Data Studio
Version: 1.41.2 (user setup)
Microsoft Windows 11 Enterprise 10.0.22621 Build 22621

### Repro Steps:

1. Launch Azure Data Studio
2. Tab till extension and search for “SQL Database Projects” and install it.
3. Tab till database project and press enter. “Database Project” panel will open.
4. Tab till 'Read our docs' link.
5. Verify whether correct role is defined for "Read our docs" as link or not.

### Actual Result:​
Incorrect role is defined for the 'read our docs' link as button.

### Expected Result:​
Correct role should be defined for the 'read our docs' control as link. As it is redirecting to new page.

### User Impact:
Screen reader user will not be able to get the proper information about the link.

### Attachment:
![Presentation1](https://user-images.githubusercontent.com/99243079/222338918-ac9ed503-3e9d-449a-90d8-9b5f317a649c.gif)

**Comment by kisantia** [comment]
same as https://github.com/microsoft/azuredatastudio/issues/22077

**Comment by Samhitha66** [comment]
@kisantia 
This bug is not same as https://github.com/microsoft/azuredatastudio/issues/22077.
The bug which you mentioned is logged with respect to MAC. The present bug is logged with respect to Win32.
We need to log different bugs for different platforms.
Thanks!

**Comment by VijayaBharathyA** [comment]
Similar issue is repro in Below screen also
### Environment and OS details:
Azure Data Studio Version: 1.42.0 (user setup)
Windows 11 Enterprise Version	22H2
Installed on	‎28-‎12-‎2022
OS build	22621.1413
Experience	Windows Feature Experience Pack 1000.22639.1000.0
### Pre-Requisite:

1. Install Azure data studio From here https://learn.microsoft.com/en-us/sql/azure-data-studio/download-azure-data-studio?view=sql-server-ver16&tabs=redhat-install%2Credhat-uninstall
2. Once installation done , Install MySQL Extension in Azure data studio
3. Tab Till  new connection and press enter 
4. Enter a valid server name username and password and press connect
### Repro Steps:

1. Launch Azure Data Studio.
2. Select the Connections icon (if not already selected) from left bar.
3. Click on the ellipses '...' and select "New Deployment"
4. In the Deployment options, select "Azure Database For MySQL" card.
5. Tab till Microsoft azure terms of use and Microsoft Privacy Statement
6. Verify that Learn more Microsoft azure terms of use and Microsoft Privacy Statement  defined as link or not
### Actual Result:
Learn more, Microsoft azure terms of use and Microsoft Privacy Statement defined as button instead of link.
### Notes:
Similar issue repro on Dashboard page also.
### Expected Result:
Learn more, Microsoft azure terms of use and Microsoft Privacy Statement should be defined as a link.
### Attachment:
![A11y_MySQLextensionforAzureDataStudio_New deployment Azure Database for MySQL_Name Role Value](https://user-images.githubusercontent.com/100189407/228875988-2559ab1f-55a3-4dd3-8236-83a151d0396e.gif)
![image](https://user-images.githubusercontent.com/100189407/233576197-0e862f4e-39ac-4c54-872b-6705bcc6b876.png)
